run-dev.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 
 ./fix-polkadot-libs.sh
+node export-env.js
```

Comment:
`yarn run dev` already have this exporting of env:
`"dev": "node export-env.js && next -p 3003",`
```suggestion
```